src/main/java/de/adesso/anki/battle/GameEngine.java

```diff
@@ -49,6 +53,38 @@ public void start() {
 
     	 
     }
+    //TODO set ready at the same time, 
+    // cool ideas like random sampling can be also used
+    
+	@Scheduled(fixedRate= 10000)
+	public void RocketReadySchedule() {
+		for (Vehicle vehicle: world.getVehicles() ) {
+			vehicle.setRocketReady(true);
+		}
+	}
+	@Scheduled(fixedRate= 10000)
+	public void MineReadySchedule() {
+		for (Vehicle vehicle: world.getVehicles() ) {
+			vehicle.setMineReady(true);
+		}
+	}
+    
+	@Scheduled(fixedRate= 10000)
+	public void ShieldSchedule() {
+		for (Vehicle vehicle: world.getVehicles() ) {
+			vehicle.setShieldReady(true);
+		}
+	}
+    
+	@Scheduled(fixedRate= 10000)
+	public void ReflectorSchedule() {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)